x/group/server/testsuite/suite.go

```diff
@@ -96,15 +96,15 @@ func (s *IntegrationTestSuite) SetupSuite() {
 		Metadata: nil,
 	})
 	s.Require().NoError(err)
-	s.groupID = groupRes.GroupId
+	s.groupID = group.ID(groupRes.GroupId)
```

Comment:
I guess we could just simply make `s.groupID` an `uint64` to avoid back and forth conversions in the tests below.